tests/test_types.py

```diff
@@ -11,45 +11,55 @@
 class TestBase:
     @dataclass
     class Config:
-        a: int
+        a: int = field(metadata=dict(positional=True))
         b: str
         c: float = 1.0
         d: int = 4
-        e: str = "e"
+        e: str = field(metadata=dict(positional=True), default="e")
```

Comment:
```suggestion
        e: str = field(default="e", metadata=dict(positional=True))
```

Reply:
Does this matter? I think it has a little value for testing purposes